sphinx/frontend-bundling.rst

```diff
@@ -0,0 +1,20 @@
+Frontend Bundling
+=================
+
+Webpack is being used to generate frontend bundles based on TypeScript code that is compiled to JavaScript as well as SCSS code which is being transpiled to CSS. The following bundles are defined:
+
+1. `editor.js/css`: Contains TinyMCE and related assets. It was decided to bundle TinyMCE in a separate file because it is the largest of all bundled assets
+2. `main.js/css`: Contains everything else
```

Comment:
```suggestion
1. ``editor.js``/``editor.css``: Contains TinyMCE and related assets. It was decided to bundle TinyMCE in a separate file because it is the largest of all bundled assets
2. ``main.js``/``main.css``: Contains everything else
```
If you're looking for a markdown-style `code` formatting, you need double back ticks in reStructuredText (single back ticks are just emphasized).